pkg/handlers/handlers.go

```diff
@@ -0,0 +1,20 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+)
+
+func Spin(http.ResponseWriter, *http.Request) {
+	fmt.Println("starting")
+	for {
+		for i := 0; i < 100000000; i++ {
+		}
+	}
+}
+
+func Unspin(http.ResponseWriter, *http.Request) {
+	fmt.Println("stopping")
+	os.Exit(200)
```

Comment:
so we just quit the program here? what happens to the pod?

Reply:
hm true. a less destructive way would be to just log `stopping` and that would be enough to end the `for` loop!

Reply:
changed it!